docs/common-pitfalls.md

```diff
@@ -63,6 +63,16 @@ test(async t => {
 
 AVA [can't trace uncaught exceptions](https://github.com/avajs/ava/issues/214) back to the test that triggered them. Callback-taking functions may lead to uncaught exceptions that can then be hard to debug. Consider promisifying and using `async`/`await`, as in the above example. This should allow AVA to catch the exception and attribute it to the correct test.
 
+### Why is the power-assert information not shown?
+
+Ensure that the first parameter passed into your test is named `t`. This is a byproduct of the way power-assert works. It uses a pattern matching scheme that makes it easier for implementors to wrap any assertion library with power-assert goodness without having to understand the ES AST at all. See [#1031](https://github.com/avajs/ava/issues/1031) for more details.
```

Comment:
Could you change this to:

> ### Why are the enhanced assertion messages not shown?
>
> Ensure that the first parameter passed into your test is named `t`. This is a requirement of how AVA uses [`power-assert`](https://github.com/power-assert-js/power-assert), the library that provides the enhanced messages. works.

I see that this paragraph originally comes from @jamestalmage's comment in https://github.com/avajs/ava/issues/1031#issuecomment-244474530. It's quite technical though. I don't think we need that kind of detail here.